web/src/main/webapp/my-app/layout/partials/app-header-options.html

```diff
@@ -26,9 +26,12 @@
         <span ng-if="GuestMode">
           <md-icon>explore</md-icon> Browse {{ portal.theme.title }}
         </span>
-        <span ng-if="!GuestMode">
+        <span ng-if="!GuestMode" hide-sm>
           <md-icon>add</md-icon> Add more to home
         </span>
+        <span ng-if="!GuestMode" hide-gt-sm>
+          <md-icon>add</md-icon> Add more
```

Comment:
Does this need some kind of ARIA treatment so that a non-visual user will understand what it is inviting the user to add more of?